doc/manual/manual.tex

```diff
@@ -176,6 +177,17 @@ \subsection{Internal Interface Arguments}
 \end{verbatim}
 }
 
+\subsection{RL-Glue Interface Arguments}
+
+\small{
+\begin{verbatim}
+  -send_rgb [true|false] -- if true, RGB values are sent for each pixel
```

Comment:
The rest of the manual uses < > for arguments (I do realize the -help printout is inconsistent with this! probably this needs to be fixed). Still, we should use < > here.